tests/python/FutureBearTest.py

```diff
@@ -0,0 +1,30 @@
+import os
+from queue import Queue
+from shutil import which
+from unittest.case import skipIf
+
+from bears.python.FutureBear import FutureBear
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+from io import StringIO
+
+
+@skipIf(which('futurize') is None, 'futurize is not installed')
+class FutureBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('test section')
+        self.uut = FutureBear(self.section, Queue())
+
+        self.test_lines = [
+            "print 'Hello World!'\n\n",
+            "d = {1: 'a', 2: 'b', 3: 'c'}\n\n",
+            "for k, v in d.iteritems():\n",
```

Comment:
This review request by gitmate is not perfect, but our tool (QuotesBear) isnt smart enough to know why double quotes should be used here.
Would you kindly switch to single quotes here, so the build passes, and we can do a human code review.